lib/src/chips_input.dart

```diff
@@ -165,6 +162,9 @@ class ChipsInputState<T> extends State<ChipsInput<T>>
   void _handleFocusChanged() {
     if (_focusNode.hasFocus) {
       _openInputConnection();
+      _suggestions = widget.initialSuggestions
```

Comment:
My gut suggests that `_suggestions` should not _repeatedly_ initialize from `widget.initialSuggestions` every time there is a focus change.

I'll need to analyze this more closely.

Reply:
I made it this way now because the value of `_suggestions` was being erased after the widget lost focus and therefore the `suggestionsBox` only appeared without the need of keyboard input on the first time the field was focused.

But I appreciate you taking your time to look into it! If you think of another solution, please let me know.